rack-dev-mark.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ['>= 2.3']
 
-  gem.add_dependency "rack", ['>= 1.1', '< 2.1']
+  gem.add_dependency "rack", ['>= 1.1', '<= 2.2.2']
```

Comment:
I think `< 2.3` is fine because a patch version change doesn't have breaking changes.